### ISSUE TITLE
Background not changing

### ISSUE THREAD

**Issue by rayzwastaken** [stepanzubkov/where-is-my-sddm-theme#15]
I could be stupid and not actually changing the config file right, but for some reason the background will not change to the specified picture. The following are what I have tried:
`background=/home/ray/Pictures/my silly cat.png`
`background=/home/ray/Pictures/"my silly cat.png"`
`background=/home/ray/Pictures/"my silly cat".png`
`background=/home/ray/Pictures/'my silly cat.png'`
`background=/home/ray/Pictures/'my silly cat'.png`
`background=/home/ray/Pictures/mysillycat.png`
`background=///home/ray/Pictures/mysillycat.png`
`background=mysillycat.png`
There are probably some others that I forgot about, almost all of these have also been done in .jpg format, and at some point of trial and error, the background just turned pure white, making all of the text unreadable. I just reinstalled the theme which fixed that. 
Anything I'm doing wrong or other things I should try?

**Comment by rayzwastaken** [comment]
Deleting everything after `backgroundFill=` (on the same line) just makes the screen completely white. The picture is just in my pictures folder, which i have permissions for, and I even tried moving the image to `/home/ray/`, which also dident work. Finnally, I also tried using another picture, and it dident work as well. This is my config after everything:
```
[General]
passwordCharacter=*
background=/home/ray/abc.png
backgroundFill=
backgroundFillMode=aspect
cursorColor=
```
PS: I am using completely AMD, which it seems like you are having other troubles with.

**Comment by rayzwastaken** [comment]
(oops i closed it on accident i havent used github in a year)
I tried a few different things, it still isnt working. I cleared `backgroundFill`, which made the screen pure white, I tried moving the image to `/home/ray/` which I definitely have permissions in, and the image has read & write permissions. I also tried changing the picture to a different picture, but still never worked. This is my config after all of this:
```
[General]
passwordCharacter=*
background=/home/ray/abc.png
backgroundFill=
backgroundFillMode=aspect
cursorColor=
```
I am using completely AMD, and I know you were having problems with AMDGPUs, could it be that? Thanks.
